package.json

```diff
@@ -18,25 +16,29 @@
     "prop-types": "^15.7.2",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
+    "webpack-dev-middleware": "^5.0.0",
+    "webpack-hot-middleware": "^2.25.0",
```

Comment:
I don't think these are dependencies.

Since this isn't hosted in the cloud, I think most-everything is actually a dependency except things that only run in local development.